PointsOfInterest/aed.csv

```diff
@@ -0,0 +1,34 @@
+name,description,note,latitude,longitude
+,Between the male and female washrooms (across from the front desk),Columbia Lake Village (CLV),43.47082,-80.56244

```

Comment:
@KartikTalwar AED stands for https://en.wikipedia.org/wiki/Automated_external_defibrillator. Would you rather that we name the file
automated_external_defibrillator.csv ?